### PR TITLE
Update the Code to Use boto 3 

### PR DESCRIPTION
**Pull request by yangky11**
Related to this [issue](https://github.com/jcjohnson/simple-amt/issues/27).

Not finished yet, but `launch_hits.py` has been fixed.

### CODE REVIEW

**Comment by yangky11** [comment]
Finished an edit pass through all files. 
Haven't tested yet.

**Comment by yangky11** [comment]
Squashed into 1 commit and ready to be merged.
Small bugs may exist but should be easy to fix later.

File: README.md [hunk 1]
```diff
@@ -173,32 +173,32 @@ To see a minimal example of these functions in action, look at the file `hit_tem
...
+- `Reward`: Required. Must be a number. This is the amount of money (in US dollars) that will be paid to workers per assignment. Keep in mind that Amazon charges the greater of 20% or 0.1 cents as a commission fee, so your actual cost per HIT will be slightly higher than the reward. You should also always pay at least 5 cents per HIT to avoid paying unnecessary commission fees to Amazon.
+- `AssignmentDurationInSeconds`: Required. Must be an integer. This is the number of time (in seconds) that each worker has to complete the HIT before it expires. You should probably make this about 2 to 3 times the actual amount of time that you expect workers to spend on each assignment.
+- `fFrame_height`: Required. Must be an integer. When you HIT is displayed, Amazon renders the HIT content inside of an iframe. The height of the iframe is `Frame_height` pixels. You should pick a number that is larger than your actual HIT content; if you don't then your HIT will be ugly and have nested scroll bars.
```
Comment (jcjohnson) [inline README.md]: Is this a typo?

File: README.md [hunk 2]
```diff
@@ -173,32 +173,32 @@ To see a minimal example of these functions in action, look at the file `hit_tem
...
+- `MaxAssignments`: Required. Must be an integer. The number of assignments to create for each input. This means that `MaxAssignments` different workers will give you results for each HIT input.
+- `Country`: Optional. Must be a string. If you set this, then only workers from the specified country will be allowed to work on your HITs. This must be either a valid [ISO 3166 country code](http://www.iso.org/iso/country_codes/country_codes) or a valid [ISO 3166-2 subdivision code](http://en.wikipedia.org/wiki/ISO_3166-2:US). I usually just use "US".
+- `Hits_approved`: Optional. Must be an integer. If you set this, then only workers who have had at least this many HITs approved on Mechanical Turk will be allowed to work on your assignments.
```
Comment (jcjohnson) [inline README.md]: Can you change to `HitsApproved` for consistent camel case on the properties file? Also same comment for the fields below.

File: approve_hits.py [hunk 3]
```diff
@@ -17,21 +18,22 @@
...
   for hit_id in hit_ids:
+    paginator = mtc.get_paginator('list_assignments_for_hit')
     try:
-      assignments = mtc.get_assignments(hit_id)
-    except:
+      for a_page in paginator.paginate(HITId=hit_id, PaginationConfig={'PageSize': 100}):
+        for a in a_page['Assignments']:
+          if a['AssignmentStatus'] == 'Submitted':
+            try:
+            # Try to parse the output from the assignment. If it isn't
```
Comment (jcjohnson) [inline approve_hits.py]: Indent the comment to match the code it describes

**Review by jcjohnson — CHANGES_REQUESTED** [review]
Looks great -- thanks for updating this! Please address a couple minor comments and I'll merge.

**Comment by yangky11** [comment]
Thanks for the comments. I have fixed them.

**Comment by jcjohnson** [comment]
Is this ready to merge now?

**Comment by yangky11** [comment]
yes

**Comment by jcjohnson** [comment]
Thanks!

**Pull request merged**